### ISSUE TITLE
Add constant given learning rate to backpropagation

### ISSUE THREAD

**Issue by jeveley** [PSZT16Z/NeuralFunc#5]

**Comment by jeveley** [comment]
Useful info: http://iamtrask.github.io/2015/07/27/python-network-part2/
